Extension/src/LanguageServer/settingsPanel.ts

```diff
@@ -21,16 +21,11 @@ const elementId: { [key: string]: string } = {
     cppStandard: "cppStandard"
 };
 
-export interface ViewStateEvent {
-    isActive: boolean;
-    isDirty: boolean;
-}
-
 export class SettingsPanel {
     private configValues: config.Configuration;
     private isIntelliSenseModeDefined: boolean = false;
-    private configDirty: boolean = false;
-    private settingsPanelViewStateChanged = new vscode.EventEmitter<ViewStateEvent>(); 
+    private settingsPanelActivated = new vscode.EventEmitter<void>();
+    private settingsPanelStateChanged = new vscode.EventEmitter<void>();

```

Comment:
Rename to onConfigValuesChanged?

Reply:
configValuesChanged?  (the other event is not using 'on')  Or, use 'on' for both?